BootloaderCommonPkg/Include/Guid/OsBootOptionGuid.h

```diff
@@ -25,6 +25,7 @@
 #define BOOT_FLAGS_CRASH_OS        BIT1
 #define BOOT_FLAGS_TRUSTY          BIT2
 #define BOOT_FLAGS_EXTRA           BIT3
```

Comment:
BOOT_FLAGS_EXTRA type does not seem to support loading images from different media devices (e.g. all images & extra images must be loaded from the same device type). Needed to add a new flag to allow loading from different media types.